src/react/mutatorMixin.js

```diff
@@ -0,0 +1,24 @@
+export const mutatorMixin = (Service, mutatingMethods) => class extends Service {
+    constructor (...args) {
+        super(...args);
+
+        this.mutatingMethods = mutatingMethods || [];
+        if (!this.mutatingMethods) {
```

Comment:
This is never true. Add .length to check if the array is empty.

Reply:
Updated array check.